CONTRIBUTING.md

```diff
@@ -27,6 +27,17 @@ Thank you for your interest in contributing to our project! <3 Whether it's a bu
 
 For a brief history and our ethos/design philosophy around the Amplify project please refer to this [document](/ETHOS.md).
 
+## Semantic versioning
```

Comment:
@harsh62  - can we make sure the semantic versioning shows up as well in the legend at the top of this page?

Reply:
@abdallahshaban557 Updated.